server/IDEs/eclipse/Refine with extensions.launch

```diff
@@ -1,18 +1,18 @@
-<booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_CLASSPATH_ONLY_JAR" value="false"/>
-<stringAttribute key="org.eclipse.jdt.launching.CLASSPATH_PROVIDER" value="org.eclipse.m2e.launchconfig.classpathProvider"/>
-<stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="com.google.refine.Refine"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="server"/>
-<stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.m2e.launchconfig.sourcepathProvider"/>
-<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Drefine.autoreloading=true -Dbutterfly.modules.path=modules,&quot;${workspace_loc:extensions}&quot;,&quot;${workspace_loc:jython}&quot;,&quot;${workspace_loc:extensions}&quot;,&quot;${workspace_loc:database}&quot;"/>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:OpenRefine}"/>
+    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
+        <listEntry value="/server/src/com/google/refine/Refine.java"/>
+    </listAttribute>
+    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
```

Comment:
Changes to this file seem unrelated with this pull request, so I would keep them out.

Reply:
Hello @wetneb. This is identical to what is already in the [repo](https://github.com/OpenRefine/OpenRefine/blob/master/server/IDEs/eclipse/Refine%20with%20extensions.launch)

Reply:
Then let us make sure this pull request does not change this file at all. You can do so by rebasing your branch on top of origin/master.

Reply:
@wetneb done

Reply:
I'm sorry, I don't understand. Was the rebase not properly done?

Reply:
Perhaps not! You could also copy over this file from the master branch and commit it to your branch.

Reply:
Ok. I have